recipes/iotaa/meta.yaml

```diff
@@ -1,28 +1,24 @@
-{% set version = "0.7.2" %}
-{% set build = "0" %}
-{% set python = "python >=3.8" %}
-
 package:
   name: iotaa
-  version: {{ version }}
```

Comment:
The version should be in the jinja template! My comment was pointing to the python version and build number only. Sorry for the confusion but when in doubt, check our docs and examples.

Reply:
Understood, will fix, thanks.

The Linux/OSX builds fail trying to run the `.exe`, so I will revert to the separate tests with selectors, unless you have another idea.

I tried adding an `rm -rf recipe`, which is fine on Linux/OSX, but which of course does not work on Windows, which I am also no expert on. Would it be bad practice to omit a Windows build for this package? If so I will find a solution.

Reply:
The idea is to not use the `.exe` and see if Windows can call the executable without it. As far as I know it is possible to run, on Windows, with both `cli` or `cli.exe`. The former will save you an extra line in the recipe and the need for the OS selector.